### PR TITLE
Fix SDK name and version sent to the Cloud Dataflow service

### PR DESCRIPTION
**Pull request by charlesccychen**

### CODE REVIEW

**Comment by charlesccychen** [comment]
R: @aaltay

File: sdks/python/apache_beam/utils/dependency.py [hunk 1]
```diff
@@ -437,6 +438,16 @@ def get_required_container_version():
     return 'beamhead'
 
 
+def get_sdk_name_and_version():
+  """Returns the name and version of the SDK reported to Cloud Dataflow."""
```
Comment (aaltay) [inline sdks/python/apache_beam/utils/dependency.py]: "Google Cloud Dataflow"

**Comment by aaltay** [comment]
LGTM, thank you.

File: sdks/python/apache_beam/utils/dependency.py (hunk 1 above)
Comment (charlesccychen) [inline sdks/python/apache_beam/utils/dependency.py]: Done.

**Comment by charlesccychen** [comment]
Thanks!

**Comment by aaltay** [comment]
R: @robertwb

**Pull request merged**